recipes/muler/meta.yaml

```
@@ -0,0 +1,57 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "muler" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ee50e30aa955d1e6a033539eb28d1dab768b2b40d32b4659eab4eb5294964a01
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python >=3.6
+
+    - astropy >=4.1
+    - specutils >=1.2
+    - numpy 
+    - scipy 
+    - celerite2
+    - pandas
+    - matplotlib
```

Unless you need a PyQt-based GUI, better to depend on matplotlib-base:
```suggestion
    - matplotlib-base
```